sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_file_system_client.py

```diff
@@ -264,6 +265,28 @@ def create_file_system(self, metadata=None,  # type: Optional[Dict[str, str]]
                                                        public_access=public_access,
                                                        **kwargs)
 
+    def create_if_not_exists(self, **kwargs):
+        # type: (Any) -> None
+        """Creates a new file system under the specified account.
+
+        If the file system with the same name already exists, it is not changed. 
+
+        :keyword Dict(str,str) metadata:
+            A dict with name-value pairs to associate with the
+            file system as metadata. Example: `{'Category':'test'}`
+        :keyword public_access:
+            To specify whether data in the file system may be accessed publicly and the level of access.
+        :type public_access: ~azure.storage.filedatalake.PublicAccess
+        :keyword int timeout:
+            The timeout parameter is expressed in seconds.
+        :rtype: None
```

Comment:
I think we talked about this but `create_file_system` says it returns a `FileSystemClient`. Is that actually true? If so, this return type of should `~azure.storage.filedatalake.FileSystemClient or None`

Reply:
I believe the answer is no, and that `create_file_system` wrongly claims to return a `FileSystemClient`. `create_file_system` returns a call into `self._container_client.create_container`, which then returns a call to `self._client.container.create`, which finally does not call into any other methods and has a return of `None, or the result of cls(response)` (verified by skimming through the code & not just looking at the docstring)

So with this finding, should I also update the docstring of `create_file_system` to correctly represent its return-type? Thanks!

Reply:
Thanks for fixing this!